.github/workflows/release-branch-tests.yml

```diff
@@ -21,6 +21,10 @@ on:
         description: 'A list of workflow(s) to run against ex: ["main.yml, "integration.yml"]'
         required: true
         type: string
+      alert_webhook:
+        description: 'Webhook url to post alerts to'
+        required: true
+        type: string
```

Comment:
I think this should be a secret so we're not publishing the webhook into our logs.  Slight change, [docs here](https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow).